PUBLISHING.md

```diff
@@ -0,0 +1,392 @@
+cargo make -e BUILDSYS_VARIANT=my-variant -e BUILDSYS_ARCH=my-arch
+```
+
+...then you can then build a repo for it like this:
+
+```shell
+cargo make repo -e BUILDSYS_VARIANT=my-variant -e BUILDSYS_ARCH=my-arch
+```
+
+## Register an AMI
+
+The [BUILDING](BUILDING.md#register-an-ami) guide covers the process of making an AMI, and has you specify `PUBLISH_REGIONS` to decide where the AMI will live.
+You can also specify this in your `Infra.toml` file:
+
+```toml
+[aws]
+regions = ["us-west-2", "us-east-1", "us-east-2"]
+```
+
+If you specify multiple regions, an AMI will be registered in the first region and then copied to the other regions.
+
+After putting this in `Infra.toml`, you can make an AMI more easily:
+
+```shell
+cargo make ami
+```
+
+If you want to change the name or description of your AMI, you can add on `-e PUBLISH_AMI_NAME=my-name` or `-e PUBLISH_AMI_DESCRIPTION=my-desc`.
+
+> Note: the AMI registration process creates a JSON file describing the AMIs in a directory under `build/images/`.
+> This file is used by the steps below when granting access to the AMIs or setting parameters in SSM.
+
+## Granting access to your AMI
+
+If you use different accounts to make and test your AMIs, you can grant access to specific accounts like this:
+
+```shell
+cargo make grant-ami -e GRANT_TO_USERS=0123456789,9876543210
+```
+
+(Later, if you need to revoke access, you can do this:)
+
+```shell
+cargo make revoke-ami -e REVOKE_FROM_USERS=0123456789,9876543210
+```
+
+## Build a repo
+
+Bottlerocket uses [TUF repositories](sources/updater/) to make system updates available to hosts.
+If you plan to update hosts rather than replace them, you'll need to make a repo.
+The repo will start out with your first image, and you can update the repo with any new images you build later, which allows your hosts to update.
+
+### Build process
+
+To build a repo, run:
+
+```shell
+cargo make repo
+```
+
+#### Picking a release time
+
+If you're preparing the release of a new version in advance (see [waves](#waves) for why you may want to) you can specify the start time for the release.
+You'll need the time in ISO 8601 format.
+You can use the `date` command to get the formatted time using a simple description of your desired start.
+For example, if you want your release to start at 10:00 AM on Monday:
+
+```shell
+RELEASE_START_TIME="$(date '+%Y-%m-%dT%H:%M:%S%:z' -d 'Monday 10am')"
+```
+
+Now we can create the repo using that time:
+
+```shell
+cargo make repo -e "RELEASE_START_TIME=${RELEASE_START_TIME}"
+```
+
+### Roles and keys
+
+#### Background on roles and keys
+
+If you run the `cargo make repo` command above without any configuration, it will generate a root role file and a signing key for you.
+The key is used to sign the metadata that lives in the repo, and the root role is included in the image and used to verify that trusted keys are being used to sign the repo in the future.
+
+The generated role and key are functional, but a bit basic.
```

Comment:
The spec itself is pretty unfriendly, so I was trying to avoid that.  We link to their official docs below, and to our updater docs above, which explain TUF and how we use it.   How do you think we could best incorporate a link in this section, and where would it point?

Reply:
I think we should maybe move the following line to the beginning of this section:
 https://github.com/bottlerocket-os/bottlerocket/blob/eb855a2de16e7f1fc16469c1de7cfaccba5a92f0/PUBLISHING.md#L276
This is where we first reference TUF metadata if users are reading from top to bottom.

In fact, maybe the whole "Expiration Policy" section could be moved upwards. Is there any reason why it's explained after the "Waves" section?

Reply:
> Is there any reason why it's explained after the "Waves" section?

It's more straightforward, and I find it useful to explain the "interesting" concepts first. :)

We'll look at some options for reorganizing this bit.